gemini/chat-completions/intro_chat_completions_api.ipynb

```diff
@@ -0,0 +1,605 @@
+      "metadata": {
+        "id": "UGokrtdiIHrX"
+      },
+      "source": [
+        "#### Gemini models\n",
+        "\n",
+        "You can experiment with [various supported Gemini models](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/call-gemini-using-openai-library#supported_models). If you are not sure which model to use then try `gemini-1.5-flash`. `gemini-1.5-flash` is optimized for multi-modal use-cases where speed and cost are important."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "r7OhyH46H2H5"
+      },
+      "outputs": [],
+      "source": [
+        "MODEL_ID = \"google/gemini-1.5-flash\"  # @param {type:\"string\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "uiWrF7U4Q5-E"
+      },
+      "source": [
+        "### [Example] Send a chat completions request\n",
+        "\n",
+        "The Chat Completions API takes a list of messages as input and returns a generated message as output. Although the message format is designed to make multi-turn conversations easy, it’s just as useful for single-turn tasks without any conversation.\n",
+        "\n",
+        "In this example, you use the Chat Completions API to send a request to the Gemini model."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "JQTWLCzoHyUA"
+      },
+      "outputs": [],
+      "source": [
+        "response = client.chat.completions.create(\n",
+        "    model=MODEL_ID, messages=[{\"role\": \"user\", \"content\": \"Why is the sky blue?\"}]\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jLNZGyOVOr1C"
+      },
+      "source": [
+        "An example Chat Completions API response looks as follows:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "DdfWBfggOvtf"
+      },
+      "outputs": [],
+      "source": [
+        "print(response)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Ipbzw5FmO5zX"
+      },
+      "source": [
+        "The generated content can be extracted with:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "LxpdxYCxH51u"
+      },
+      "outputs": [],
+      "source": [
+        "response.choices[0].message.content"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nSIVXw7bMr4F"
+      },
+      "source": [
+        "You can use `Markdown` to display the formatted text."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "zOwEvNkCMiMF"
+      },
+      "outputs": [],
+      "source": [
+        "from IPython.display import Markdown\n",
```

Comment:
Should we move the imports to the top of the notebook?

Reply:
This is an optional setup for markdown, and it's only used for this cell. May be it's better can be kept in this cell.